README.md

```diff
@@ -780,6 +780,9 @@ Sensors, 2022. [PDF](https://arxiv.org/pdf/2207.04007),  [Project page](https://
 *[Secrets of Event-based Optical Flow](https://arxiv.org/pdf/2207.10022)*,  
 Europen Conf. Computer Vision (ECCV), 2022. [YouTube](https://youtu.be/nUb2ZRPdbWk), [Poster](https://drive.google.com/file/d/1mF-mM4teb8A9bKJJiQwN7IFsGsRIsRaX/view?usp=sharing), [Presentation at  the  PRG  Seminar Series  U. Maryland (Video)](https://vid.umd.edu/detsmediasite/Play/d31926745bdd446e8f6ce165ae4811591d), [Presentation at the GRASP Laboratory (UPenn) seminar](https://youtu.be/8v-bYCVL9hw),
 [Project page and Code](https://github.com/tub-rip/event_based_optical_flow)
+- <a name="Shiba22spl"></a>Shiba, S., Aoki, Y., Gallego, G.,  
+*[Fast Event-Based Optical Flow Estimation by Triplet Matching](https://ieeexplore.ieee.org/document/10008035)*,
```

Comment:
![image](https://github.com/uzh-rpg/event-based_vision_resources/assets/8024432/20baf786-92b1-49c7-90e2-8a9251b1e01d)

Need a line break.
Authors, title and publication venue should be on 3 separate lines.

Reply:
Sorry, I didn't realize the whitespaces at the end of the title line.